src/App.js

```diff
@@ -1,12 +1,16 @@
 import {
-  BrowserRouter,
+  BrowserRouter, Routes, Route,
 } from 'react-router-dom';
 import NavBar from './components/NavBar';
+import RocketsList from './features/rockets/RocketsList';
 
 function App() {
   return (
     <BrowserRouter>
       <NavBar />
+      <Routes>
+        <Route path="/Rockets" element={<RocketsList />} />
+      </Routes>
     </BrowserRouter>
   );
```

Comment:
I can't see the other components in the App.js while this had been approved and merged to dev and it had been agreed to create the new branch from that dev branch. Would you please merge your branch with dev (not the pull request) so that I have a larger view and so that we don't have to deal with those conflicts anymore?